src/main/java/org/jenkinsci/plugins/tuleap_git_branch_source/TuleapSCMSource.java

```diff
@@ -163,6 +163,9 @@ protected void retrieve(@CheckForNull SCMSourceCriteria criteria, @NonNull SCMHe
                 }
 
             }
+            if(request.isRetrievePullRequests()){
+                request.listener().getLogger().format("Retrieving pull request for repository at %s %n", repositoryPath);
```

Comment:
I'm not sure what the `%n` is doing here ?

Reply:
The `%n` is needed to avoid to have the next printed log on the same line.